Dockerfile

```diff
@@ -1,23 +1,49 @@
-FROM openmicroscopy/ome-files-cpp-u1604
+FROM sbesson/ome-files-cpp-u1604
 MAINTAINER ome-devel@lists.openmicroscopy.org.uk
 
-RUN apt-get update && apt-get -y install \
-  default-jdk \
-  maven
+# Install JDK7 and Maven
+RUN apt-get -y install software-properties-common
+RUN add-apt-repository ppa:openjdk-r/ppa
+RUN apt-get update
+RUN apt-get -y install openjdk-7-jdk openjdk-7-jre-lib
+RUN apt-get -y install maven
+ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
 
-ADD . /git/ome-files-performance
+#  Bio-Formats JACE bindings
+RUN git clone https://github.com/ome/bio-formats-jace /git/bio-formats-jace
+WORKDIR /git/bio-formats-jace
+RUN mvn -DskipTests clean package cppwrap:wrap dependency:copy-dependencies
 
-WORKDIR /build2
+COPY cmake/JACEPrerequisites.cmake /git/bio-formats-jace/target/cppwrap
```

Comment:
Needs renaming as well; the target is `/git/bio-formats-jace/target/cppwrap/Prerequisites.cmake`?